packages/grafana-data/src/field/fieldOverrides.test.ts

```diff
@@ -118,6 +118,7 @@ describe('applyFieldOverrides', () => {
           overrides: [],
         },
         replaceVariables: (value: any) => value,
+        getDataSourceSettingsByUid: undefined as any,
```

Comment:
I assume we are going to remove this in the refactor of `getLinks`?

Reply:
Yes, right now I do not see any easy way around this, but as we talked, this should be refactored into a (hopefully) more straightforward processing without the need to pass all these functions around.